pages/common/pdfimages.md

```diff
@@ -0,0 +1,20 @@
+# pdfimages
+
+> Utility for extracting images from PDFs.
+
+- Extract all images from a PDF file and save them in PNG format:
+
+`pdfimages -png {{PDF-file}} {{image-root}}`
+
+- Extract images from a PDF file from pages 3-5:
+
+`pdfimages -f 3 -l 5 {{PDF-file}} {{image-root}}`
+
+- Extract images from a PDF file and include the page number in their names:
```

Comment:
Perhaps this will read better?
```suggestion
- Extract images from a PDF file and include the page number in the output filenames:
```

Reply:
I disagree with you on this one, but :+1: for the other suggestions.

Reply:
Why do you disagree/

Reply:
Oh... I thought it was related to PDF files and not the generated files. :+1: for this one too.